openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedEditor/SendButton.tsx

```diff
@@ -11,72 +11,34 @@
  *  limitations under the License.
  */
 
-import { Popover } from 'antd';
+import { Button } from 'antd';
 import classNames from 'classnames';
-import React, { FC, Fragment } from 'react';
-import { useTranslation } from 'react-i18next';
-import { Transi18next } from '../../../utils/CommonUtils';
-import SVGIcons, { Icons } from '../../../utils/SvgUtils';
-import { Button } from '../../buttons/Button/Button';
+import React, { FC } from 'react';
+
+import { ReactComponent as IconPaperPlanePrimary } from '../../../assets/svg/paper-plane-primary.svg';
 
 interface SendButtonProp {
   editorValue: string;
   buttonClass: string;
   onSaveHandler: () => void;
 }
 
-const getIcon = (editorValue: string) => {
-  return editorValue.length > 0 ? Icons.PAPER_PLANE_PRIMARY : Icons.PAPER_PLANE;
-};
-
```

Comment:
@Ashish8689 , Why did we remove this logic?

Reply:
@Sachin-chaurasiya This logic is for the icon change when text is not there and we are also disabling the button too. So just the primary button does our job.